ci-operator/step-registry/openshift/ci-security/snyk/scan/openshift-ci-security-snyk-scan-commands.sh

```diff
@@ -13,6 +13,16 @@ chmod +x ${SNYK_DIR}/snyk
 echo snyk installed to ${SNYK_DIR}
 ${SNYK_DIR}/snyk --version
 
+# install jq if not installed
+if ! [ -x "$(command -v jq)" ]; then
+    JQ_DIR=/tmp/jq
```

Comment:
I think you can just put it to ${SNYK_DIR} for simplicity

Reply:
updated